build.sbt

```diff
@@ -78,10 +79,18 @@ val samzaDependencies = Seq(
   "org.apache.samza" % "samza-api" % samzaVersion % "optional"
 )
 
+val sparkDependencies = Seq (
+  "org.apache.spark" % "spark-streaming_2.10" % sparkVersion % "optional"
```

Comment:
Perhaps try:

```scala
"org.apache.spark" %% "spark-streaming" % sparkVersion % "optional"
```

The %% will make sbt pull in the appropriate 2.10 or 2.11 version depending on what we're building with.